addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1898,6 +1971,57 @@ var SnippetsMenu = Widget.extend({
             $snippet.toggleClass('o_disabled', !check);
         });
     },
+    /**
+     * Gets $element css animations and transitions max duration.
+     *
+     * @private
+     * @returns {Number} max duration (Integer) of the animations and transitions for the overlay target
+     */
+    _getElementMaxAnimationAndTransitionDuration: function ($element) {
+        const cssDurationProperties = ['animation-duration', 'transition-duration'];
+        let maxDuration = 0;
+        for (let index = 0, len = cssDurationProperties.length; index < len; index++) {
+            const key = cssDurationProperties[index];
+            const compStyle = window.getComputedStyle($element.get(0), null);
+            const durationProperty = compStyle.getPropertyValue(key);
+            // Safari returns null and typeof object, but Chrome returns 0s
+            if (durationProperty && durationProperty !== '0s') {
+                const transItems = durationProperty.toLowerCase().split(',');
+                // loop through transition values
+                for (let j = 0; j < transItems.length; j++) {
+                    let duration = transItems[j];
+                    if (duration.indexOf('ms') !== -1) {
+                        // time value in milliseconds
+                        duration = duration.replace('ms', '');
+                        duration = Math.ceil(parseFloat(duration));
+                    } else {
+                        // time value in seconds, convert to milliseconds
+                        duration = duration.replace('s', '');
+                        duration = Math.ceil(parseFloat(duration) * 1000);
+                    }
+                    if (duration > maxDuration) {
+                        maxDuration = duration;
+                    }
+                }
+            }
+        }
+        return maxDuration;
+    },
+    /**
+     * Gets $element positional id towards #wrap div, optionally appended with a provided child positional id.
+     *
+     * @private
+     * @param {jQuery} $element
+     * @param {String} [childPositionalId=''] child positional id to append
+     * @returns {String} the positional id of $element towards #wrap div appended with the provided child positional id if any
+     */
+    _getPositionalId: function ($element, childPositionalId = '') {
+        if (!$element.is('div#wrap')) {
```

Comment:
I think it should be "div#wrapwrap" because you could add an animation on elements outside the wrap div .
That way you get an infinite loop when you add an animation on the footer.

Reply:
👍

Reply:
The editor is not necessarily working with #wrapwrap elements, not related to the editor ;) I'll find a way